docs/design/mockups/phase-6-team/6cd4-squadgrid/round-1-playercard.html

```
@@ -0,0 +1,114 @@
+    .single { max-width: 720px; margin: 16px auto 0; background: var(--color-cream); border: 2px dashed var(--color-ink-muted); padding: 18px; display:flex; gap: 22px; flex-wrap:wrap; }
+    .single .col { width: 150px; }
+    .single .lbl { font-family: var(--font-mono); font-size:10px; text-transform:uppercase; letter-spacing:.08em; color:var(--color-ink-muted); margin-bottom:8px; }
+  </style>
+</head>
+<body>
+
+  <div class="drill-head">
+    <div class="tag">Phase 6.C · Team detail · Round 1 · Drill 6.C.d4 (SquadGrid / PlayerCard)</div>
+    <h1>The squad grid + the player card.</h1>
+    <p>Grouped by position (Doelmannen / Verdedigers / Middenvelders /
+       Aanvallers). Each card composes the locked `<PlayerFigure>` (newsprint
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </playerfigure> ], start tag match failed [ <PlayerFigure> ] on line 70.

(tag-pair)